vignettes/references.bib

```diff
@@ -11,4 +11,145 @@ @Article{Isaksen2023
+  volume  = {57},
+  doi     = {10.1007/s00125-014-3225-9},
+  type    = {Journal Article},
+}
+
+@Article{Bak2021,
+  author  = {Bak, Jessica C. G. and Serné, Erik H. and Kramer, Mark H. H. and Nieuwdorp, Max and Verheugt, Carianne L.},
+  journal = {Acta Diabetologica},
+  title   = {National diabetes registries: do they make a difference?},
+  year    = {2021},
+  issn    = {1432-5233},
+  number  = {3},
+  pages   = {267-278},
+  volume  = {58},
+  doi     = {10.1007/s00592-020-01576-8},
+  type    = {Journal Article},
+}
+
+@Article{HallgrenElfgren2016,
+  author  = {Hallgren Elfgren, Ing-Marie and Grodzinsky, Ewa and Törnvall, Eva},
+  journal = {Primary Health Care Research &amp; Development},
+  title   = {The Swedish National Diabetes Register in clinical practice and evaluation in primary health care},
+  year    = {2016},
+  issn    = {1463-4236},
+  number  = {6},
+  pages   = {549-558},
+  volume  = {17},
+  doi     = {10.1017/S1463423616000098},
+  type    = {Journal Article},
+  url     = {https://www.cambridge.org/core/article/swedish-national-diabetes-register-in-clinical-practice-and-evaluation-in-primary-health-care/F3A9B74FED1E7B36147FF63382939B9A},
+}
+
+@Article{Cooper2013,
+  author  = {Cooper, John G. and Thue, Geir and Claudi, Tor and Løvaas, Karianne and Carlsen, Siri and Sandberg, Sverre},
+  journal = {Norsk Epidemiologi},
+  title   = {The Norwegian Diabetes Register for Adults – an overview of the first years},
+  year    = {2013},
+  number  = {1},
+  volume  = {23},
+  doi     = {10.5324/nje.v23i1.1599},
+  type    = {Journal Article},
+  url     = {https://www.ntnu.no/ojs/index.php/norepid/article/view/1599},
+}
+
+
+
+@Article{Carstensen2011,
+  author  = {Carstensen, B. and Kristensen, J. K. and Marcussen, M. M. and Borch-Johnsen, K.},
+  journal = {Scand J Public Health},
+  title   = {The National Diabetes Register},
+  year    = {2011},
+  issn    = {1403-4948},
+  number  = {7 Suppl},
+  pages   = {58-61},
+  volume  = {39},
+  doi     = {10.1177/1403494811404278},
+  type    = {Journal Article},
+}
+
+@Article{Green2014,
+  author  = {Green, Anders and Sortsø, Camilla and Jensen, Peter Bjødstrup and Emneus, Martha},
+  journal = {Clinical epidemiology},
+  title   = {Validation of the Danish National Diabetes Register},
+  year    = {2014},
+  issn    = {1179-1349},
+  pages   = {5-15},
+  volume  = {7},
+  doi     = {10.2147/CLEP.S72768},
+  type    = {Journal Article},
+  url     = {https://www.ncbi.nlm.nih.gov/pubmed/25565889
+https://www.ncbi.nlm.nih.gov/pmc/PMC4274151/},
+}
+
+@Misc{DHDA2016,
+  author = {{Danish Health Data Authority}},
+  title  = {{Algoritmer for udvalgte kroniske sygdomme og svære psykiske lidelser [English: Algorithms for selected chronic diseases and serious mental illnesses]}},
+  year   = {2016},
+  url    = {https://www.esundhed.dk/-/media/Files/Dokumentation/Register-for-Udvalgte-Kroniske-Sygdomme-og-Svaere-Psykiske-Lidelser/29_Algoritmer_for_RUKS.ashx},
+}
```

Comment:
The links to previous RSCD/RUKS documentation are killed every time they update the documentation, so the links to DHDA2016 and SDHDA2022lastaccessed7Feb2023 are now dead. The original documentation is inaccessible to the public and all you have to go by is the "revision log" table in the current word document, which is not really sufficient for reproducibility.